hnet/writers.go

```diff
@@ -106,6 +106,17 @@ func (request SpectateRequest) Serialize(stream *common.IOStream) {
 	stream.WriteU32(request.UserId)
 }
 
+func (request HasMapRequest) Serialize(stream *common.IOStream) {
+	stream.WriteU8(1)
+	stream.WriteU32Bool(request.HasMap)
+}
+
+func (response HasMapResponse) Serialize(stream *common.IOStream) {
+	stream.WriteU32(2)
```

Comment:
Why is this 2 specifically?

Reply:
Hexis reads the user id and boolean as a list, so there is 2 elements

Reply:
Wow, this explains a *lot*...